rules/falco_rules.yaml

```diff
@@ -3052,10 +3052,38 @@
 - list: c2_server_ip_list
   items: []
 
-- rule: Outbound Connection to C2 Servers
-  desc: Detect outbound connection to command & control servers
-  condition: outbound and fd.sip in (c2_server_ip_list)
-  output: Outbound connection to C2 server (command=%proc.cmdline pid=%proc.pid connection=%fd.name user=%user.name user_loginuid=%user.loginuid container_id=%container.id image=%container.image.repository)
+- list: c2_server_fqdn_list
+  items: []
+
+# Use Falco HELM chart to update (append) the c2 server lists with your values.
+#
+#  ```yaml
+#  [...]
+#  customRules:
+#    c2-servers-list.yaml: |-
+#      - list: c2_server_ip_list
+#        append: true
+#        items: 
+#        - "'51.178.161.32'"
+#        - "'46.101.90.205'"
+#        
+#      - list: c2_server_fqdn_list
+#        append: true
+#        items: 
+#        - "srv-web.ffconsulting.com"
+#        - "57.ip-142-44-247.net"
+#  ```
+
+- rule: Outbound Connection to C2 Servers IPs and FQDNs
```

Comment:
I honestly don't think we need to change the rule name since it's accurate as it is. 
we can just add in the existent description that the rules match IPs and FQDN.
I would just update the rule condition as done to support fqdn as well

Reply:
Hi, I agree  with your suggestions. The name of the rule can stay the same (`Outbound Connection to C2 Servers`), as well as the `output:` field.

If you prefer, I can revert to the previous rule name.

***

Question: I understand you prefer this

```yaml
  desc: Detect outbound connection to command & control servers thanks to a list of IP addresses & a list of FQDN.
```

over this?

```yaml
  desc: >
    Detect outbound connection to command & control servers. For example, fetch
    a list of IP addresses and FQDN on this website:
    https://feodotracker.abuse.ch/downloads/ipblocklist_recommended.json.
```

I have no opinion on this one. I understand it is maybe better not to include a link to `feodotracker.abuse.ch`.

Reply:
@Nicolas-Peiffer I'll go with this one
`  desc: Detect outbound connection to command & control servers thanks to a list of IP addresses & a list of FQDN.
`
I feel the desc you added is more a comment for the rule so since it is actually a useful comment I think you can add it on top fo the rule as a proper comment as done in other rules. What do you think?